nats.go

```diff
@@ -2507,6 +2507,19 @@ func (nc *Conn) IsConnected() bool {
 	return nc.isConnected()
 }
 
+// Servers returns the list of known servers, including additional
```

Comment:
Question to @derekcollison what difference it makes? The user adds server urls (with credentials) in Options.Servers. The user can call Options.Servers and will see those. The Conn.Servers returns Options.Servers + the one that it got from INFO. *Those* don't contain credentials.